build_tools/android/post_benchmarks_as_pr_comment.py

```diff
@@ -299,18 +302,25 @@ def get_benchmark_result_markdown(benchmark_files: Sequence[str],
   pr_commit = get_required_env_var("BUILDKITE_COMMIT")
   pr_commit = md.link(pr_commit,
                       f"{GITHUB_IREE_REPO_PREFIX}/commit/{pr_commit}")
+
+  commit_info = f"@ commit {pr_commit}"
   if query_base:
-    # Update the aggregate benchmarks with base numbers.
-    base_commit = get_origin_tree_top_commit(verbose)
-    base_benchmarks = query_base_benchmark_results(base_commit, verbose)
-    for bench in base_benchmarks:
-      if bench in all_benchmarks:
-        all_benchmarks[bench].base_mean_time = base_benchmarks[bench]
-    base_commit = md.link(base_commit,
-                          f"{GITHUB_IREE_REPO_PREFIX}/commit/{base_commit}")
-    commit_info = f"@ commit {pr_commit} (vs. base {base_commit})"
-  else:
-    commit_info = f"@ commit {pr_commit}"
+    # Try to query some base benchmark to diff against, from the top of the
+    # tree. Bail out if the maximal trial number is exceeded.
```

Comment:
Maybe add a note if no base benchmark is found (so we can at least spot where this fails more easily)?

Reply:
https://github.com/google/iree/pull/6767